gammapy/scripts/analysis.py

```diff
@@ -0,0 +1,55 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import click
+import logging
+from gammapy.analysis import Analysis, AnalysisConfig
+
+log = logging.getLogger(__name__)
+
+
+@click.command(name="config")
+@click.option(
+    "--filename",
+    default="config.yaml",
+    help="Filename to store the default configuration values.",
+    show_default=True
+)
+@click.option(
+    "--overwrite",
+    default=False,
+    is_flag=True,
+    help="Overwrite existing file."
+)
+def cli_make_config(filename, overwrite):
+    """Writes default configuration file."""
+    config = AnalysisConfig.from_template("1d")
```

Comment:
Please use `config = AnalysisConfig()`. I want to remove `from_template` now. There's no reason why `from_template("1d")` is better than the default, no?

Reply:
Ok. 
But I've added it to the tests, since I need a config file with proper default that do not break at `get_datasets`. You'll remove it from the tests in the PR that deletes the templates.